CHANGELOG.rst

```diff
@@ -23,6 +23,7 @@ Documentation:
 Internal changes:
 
 - Do not scrub versions in reference data. (:issue:`747`)
+- Add interface for the different formats to easily add new formats. (:issue:`755`)
```

Comment:
The Changelog does not yet mention the new CLI option aliases like `--txt-summary`.